tracer/test/Datadog.Trace.Security.IntegrationTests/AspNetBase.cs

```diff
@@ -77,6 +76,14 @@ public override void Dispose()
             _httpClient?.Dispose();
         }
 
+        public void AddHeaders(Dictionary<string, string> headersValues)
+        {
+            foreach (var header in headersValues)
+            {
+                _httpClient.DefaultRequestHeaders.Add(header.Key, header.Value);
```

Comment:
nit: I expect this gives a warning on .NET 8, because calling `Add` will throw if you try to add a duplicate header. I think you can stop it complaining by doing this (not tested)?

```suggestion
                _httpClient.DefaultRequestHeaders[header.Key] = header.Value;
```

Reply:
Looks like HttpRequestHeaders doesn't support indexing. Anyway, no warning seems to be shown in that line.